lib/widgets/rules_buttons.dart

```diff
@@ -0,0 +1,124 @@
+import 'package:flutter/material.dart';
+
+import 'game_view.dart';
+
+class RulesChangingButtons extends StatefulWidget {
+  const RulesChangingButtons({super.key});
+
+  @override
+  RulesChangingButtonsState createState() => RulesChangingButtonsState();
+}
+
+class RulesChangingButtonsState extends State<RulesChangingButtons> {
+  //int rules = 0;
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.center,
+      children: [
+        const Text('Get'),
+        const SizedBox(width: 20),
+        SizedBox(
+          width: 87,
+          height: 30, // Replace with the desired width
+          child: OutlinedButton(
```

Comment:
Why we use button to wrap another button?

Reply:
easier to change size

Reply:
flutter says that better to replace Container with SizedBox

Reply:
I still see OutlinedButton in the code.
Container can do both size and border.
Please, do it for both drop downs.

Reply:
Now its made another way